extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -413,6 +421,21 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
     return googleLifecycleReporterFactory(this);
   }
 
+  /** @override */
+  isXhrAllowed() {
+    if (!isCanonical(this.win)) {
```

Comment:
Why do you need to get the experiment here at all given you wouldn't have executed FF if you're canonical and in the control (or exp not selected).  Can't this just be:  isXhrAllowed() { return !isCanonical(this.win); }